src/sentry/services/hybrid_cloud/integration/impl.py

```diff
@@ -210,44 +211,29 @@ def get_organization_integrations(
 
         return [serialize_organization_integration(oi) for oi in ois]
 
-    def get_organization_context(
-        self,
-        *,
-        organization_id: int,
-        integration_id: int | None = None,
-        provider: str | None = None,
-        external_id: str | None = None,
-    ) -> tuple[RpcIntegration | None, RpcOrganizationIntegration | None]:
-        integration, installs = self.get_organization_contexts(
-            organization_id=organization_id,
-            integration_id=integration_id,
-            provider=provider,
-            external_id=external_id,
-        )
-
-        return integration, installs[0] if installs else None
-
     def get_organization_contexts(
         self,
         *,
         organization_id: int | None = None,
         integration_id: int | None = None,
         provider: str | None = None,
         external_id: str | None = None,
-    ) -> tuple[RpcIntegration | None, list[RpcOrganizationIntegration]]:
+    ) -> RpcOrganizationIntegrationContextResult:
```

Comment:
This looks like a breaking change? Changing return types requires multiple deploys to do safely.

1. Add a new method with the updated return type. It will need a new name unfortunately.
2. Start using the new method.
3. Update the old method return type once it is no longer called.
4. Move usage back to the original method.
5. Remove the compatibility shim method.